package.json

```diff
@@ -11,7 +11,25 @@
     "acorn.js",
     "interpreter.js",
     "LICENSE",
-    "README.md"
+    "README.md",
+    "bin"
   ],
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "bin": "bin/run.js",
+  "resolutions": {
+    "eshost": "pcardune/eshost#f3fa7fa",
```

Comment:
I've opened an issue which would obviate the need for this forked version of eshost: https://github.com/bterlson/test262-harness/issues/71

Hopefully it will get looked at sometime soon.